recipes/raven-hydro/build.sh

```diff
@@ -0,0 +1,8 @@
+# (1) Replace "g++" by $(CXX) in the Makefile
+sed -i 's/CXX      \:\= g++/# CXX      \:\= g++/g' ./Makefile
+# (2) Rename the app to Raven, not Raven.exe in the Makefile
+sed -i 's/appname \:\= Raven\.exe/appname \:\= raven/g' Makefile
+
+make
+
```

Comment:
Is there a make check or something similar?

Reply:
No, there aren't any recipes aside from the make build call and a `make clean`